### PR TITLE
10.Customizing-Lock

### PR DESCRIPTION
**Pull request by volbap**
- Added `10-customizing-lock.md` tutorial, pretty much based on [this document](https://github.com/auth0/docs/blob/master/articles/libraries/lock-ios/customization.md). 
- Updated `index.yml`.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ios-swift/10-customizing-lock.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+let myAwesomeTheme = A0Theme()
+// Configure your theme here (see step 2)
+A0Theme.sharedInstance().registerTheme(myAwesomeTheme)
+```
+
+### 2. Configure your Theme
+
+Lock's widget UI is composed of several parts that can be customized. Here is a list of the available values that you can modify:
+
+![Lock.png](https://github.com/auth0/docs/blob/master/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/10-customizing-lock.md]: @pablolvillar Displays as broken image https://auth0-docs-content-pr-1773.herokuapp.com/docs/quickstart/native/ios-swift/10-customizing-lock

image

It should be: /media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png

**Comment by volbap** [comment]
@tenmaster this is pending review

File: articles/native-platforms/ios-swift/10-customizing-lock.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+- `A0ThemeCredentialBoxBorderColor`
+- `A0ThemeCredentialBoxSeparatorColor`
+- `A0ThemeCredentialBoxBackgroundColor`
+
+#### Close Button
+
+- `A0ThemeCloseButtonTintColor`
+
+Have fun customizing whatever you need from there.
```
Comment (tenmaster) [inline articles/native-platforms/ios-swift/10-customizing-lock.md]: from here... if you refer to all those properties.

**Comment by tenmaster** [comment]
+1 @mpaktiti

**Comment by tenmaster** [comment]
+1 @ntotten too :)

**Comment by mpaktiti** [comment]
@pablolvillar Please resize the /media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png image, it currently is 1620 × 1334. It should be maximum 750 pixels wide.
Also, rebase to resolve conflicts.

**Comment by volbap** [comment]
@mpaktiti done

**Comment by mpaktiti** [comment]
Seed project (to be deployed before this PR): https://github.com/auth0-samples/auth0-ios-swift-sample/pull/10

**Comment by mpaktiti** [comment]
@pablolvillar Please review the comments and apply the fixes. Thank you.

**Comment by volbap** [comment]
@mpaktiti Done with the fixes.

Notice that the image has been resized here but I didn't add the 1px border (I just did that in another branch, and I assume that given this change is older, the other branch would overwrite it when merging and leave the proper resized+bordered image)

**Comment by mpaktiti** [comment]
Before deployment:
- [Seed project](https://github.com/auth0-samples/auth0-ios-swift-sample/pull/10) should be merged. The link to this doc should be updated.
- Sample is at auth0-samples repo, download zip functionality should be in place.
- `index.yml` should be checked again

File: articles/native-platforms/ios-swift/10-customizing-lock.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+let myAwesomeTheme = A0Theme()
+// Configure your theme here (see step 2)
+A0Theme.sharedInstance().registerTheme(myAwesomeTheme)
+```
+
+### 2. Configure your Theme
+
+Lock's widget UI is composed of several parts that can be customized. Here is a list of the available values that you can modify:
+
+![Lock.png](/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/10-customizing-lock.md]: This image is too wide, it get's fixed though by [this PR](https://github.com/auth0/docs/pull/1789).

**Comment by volbap** [comment]
Closing this pull request. These changes will be included in a new pull request.

**Pull request closed without merging**